frontend/src/components/text/ImageHandler.tsx

```
@@ -0,0 +1,124 @@
+            props: {
+              handleDrop: (view, event) => {
+                let isImage = false;
+                const file = event.dataTransfer?.files[0];
+
+                let auditId: string | null = null;
+                const path = window.location.pathname.split('/');
+                if (path && path.length > 3 && path[1] === 'audits') {
+                  auditId = path[2];
+                }
+
+                if (file && file.type.startsWith('image')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)